spec/features/super_admin/logging/view_logs_for_ip_spec.rb

```diff
@@ -0,0 +1,73 @@
+describe 'View authentication requests for an IP', type: :feature do
+  let(:user) { create(:user) }
+  let(:user_location) { create(:location, organisation: user.organisation) }
+  let(:ip_1) { create(:ip, location_id: user_location.id, address: '1.2.3.4') }
```

Comment:
Use ActiveRecords association helpers, not ids. Eg: `location: user_location` not `location_id: user_location.id`

Reply:
Also worth checking other places you've used it in this spec